usaspending_api/etl/management/sql/c_file_linkage/update_file_c_linkages_fain.sql

```diff
@@ -14,16 +14,20 @@ WHERE
     faba.financial_accounts_by_awards_id = ANY(
         SELECT
             faba_sub.financial_accounts_by_awards_id
-        FROM
-            financial_accounts_by_awards AS faba_sub
+        FROM (
+            SELECT fain, uri, award_id, financial_accounts_by_awards_id, submission_id
+            FROM financial_accounts_by_awards AS faba_sub
+        ) AS faba_sub
+        JOIN (
+            SELECT fain
+            FROM {file_d_table} as aw_sub
+            GROUP BY fain
+            HAVING count(*) = 1
+        ) AS aw_sub
+        ON aw_sub.fain = faba_sub.fain
```

Comment:
Looks like the `UPPER()` function was removed from this comparison (same for other queries). We talked about doing this, but if we do, we'll need to take a few more steps. 

1. We've already confirmed that string fields on the `financial_accounts_by_awards` table are being uppercased in the Submission Loader, but @aguest-kc did find that we have a few records with non uppercased fields from 2018. A data change ticket will be needed to correct this. 
2. We need to perform the same analysis on the Awards table? Are there any fields with lowercase values? If not, is there a way to guarantee that this will always be the case? ex: Is there an upstream step that is capitalizing these values?

Reply:
Why not just add the `UPPER()` function back and leave it at that? I think it was removed on accident.

Reply:
I confirmed with Daniel that the Broker does accept lowercased values for these fields, because they want to keep the data in the same format that it was submitted in. He also thinks that we already uppercase the data during the pipeline.

I think we should fix the few rows that are currently lowercased, but also add the `UPPER()` calls back in just to be sure we're comparing the data in the same format.

Reply:
It was definitely removed by accident, here is the original

`WHERE UPPER(aw_sub.fain) = UPPER(faba_sub.fain)`

We should just add it back.

Reply:
We uppercase these values as part of the pipeline, so these `UPPER()` calls aren't necessary. I'm creating a data change ticket to correct some old rows that do have lowercase values in the FABA table, but then everything will be in uppercase format.

Reply:
I am going to validate performance improvements by not having this UPPER. If there are no performance improvement, I don't see this being within the scope of this story.